changelog.md

```diff
@@ -1,5 +1,61 @@
 # Changelog
 
+
+## 0.15.0
+
+This release is brings download progress bars for file downloads
+and does an overhaul on printing.
+
+* TODO add commits merged after 4ca54cc
+
+* Initial support for IPP / driverless printing #2332
+* Use Breaks to remove old print-related dependencies #2434
+* Replace XPP with GTK print dialog #2411
+* Display download progress for file downloads
```

Comment:
We can just condense all the progress bar stuff into one line with multiple PR numbers, since all those PRs are just follow-ups to the original one, and not new changes.